engine/baml-runtime/src/runtime/runtime_interface.rs

```diff
@@ -368,6 +416,7 @@ impl RuntimeConstructor for InternalBamlRuntime {
 
         Ok(Self {
             ir: Arc::new(ir),
+            db: schema.db,
```

Comment:
It feels unfortunate that the runtime would carry the parser database around - it's a little bit of a scrambling of the compiler passes. But I don't know if there's an easy way to avoid this.